src/PageLayout/Notification/NotificationDetailsController.js

```diff
@@ -0,0 +1,217 @@
+                donatingAgency: '-K9HdKlCLjjk_ka82K0s',  // autogen-key of a donating-agency
+                requester: 'Andrea Benson',  // name of a donating-agency-member
+                receivingAgency: {
+                    // Depending on the status of the request, this
+                    // map will have 1 out of the 3 possible fields:
+                    // "requested", "pending" (list), "claimed"
+                    requested: 'uGOFJ8NqHjbZhKAYzSZFRs1dSKD3',  // uid-key of a RA
+                    // OR
+                    pending: [  
+                        'uGOFJ8NqHjbZhKAYzSZFRs1dSKD3',  // uid-key of all RAs
+                    ],
+                    // OR
+                    // claimed: 'uGOFJ8NqHjbZhKAYzSZFRs1dSKD3',  // uid-key of a RA (once a RA claims)
+                },
+                // ADDED FOR DUMMY DATA
+                daInfo: {
+                    name: 'Local Point',
+                    primaryContact: { // primaryContact of the request not the DA
+                        name: 'Andrea Benson',
+                        email: 'AndreaIsCool@uw.edu',
+                        phone: '206-586-9876',
+                        position: 'Manager'
+                    },
+                    address: {
+                        street1: '1201 NE Campus Pkwy',
+                        street2: '',
+                        city: 'Seattle',
+                        state: 'WA',
+                        zipcode: 98105,
+                        officeNo: '220'
+                    }
+                },
+                raInfo: {
+                    // ADDED FOR DUMMY DATA
+                    name: 'Seattle Gospel Union Mission',
+                    address: {
+                        street1: '3802 South Othello Street',
+                        street2: '',
+                        city: 'Seattle',
+                        state: 'WA',
+                        zipcode: 98118,
+                        officeNo: ''
+                    },
+                    primaryContact: {
+                        name: 'Amy Powell',
+                        email: 'amy.powell@uw.edu',
+                        phone: '206-333-2343',
+                        position: 'Manager'
+                    }
+                },
+                delivererGroup: {
+                    claimed: 'R8BAHrxdkfQoAmfWvGa1OJmjQP43',  // uid-key of a DG (once a DG claims)
+                    // ADDED FOR DUMMY DATA
+                    deliverers: [
+                        // {
+                        //     name: 'Alice',
+                        //     email: 'alice@uw.edu',
+                        //     phone: '123-789-4560'
+                        // },
+                        // {
+                        //     name: 'Chris',
+                        //     email: 'chris@uw.edu',
+                        //     phone: '456-123-0789'
+                        // }
+                    ]
+                },
+                dgInfo:{
```

Comment:
(see comment in `RequestSummary` about what to show for DG) Instead of `deliverers`, fill in the same info as `raInfo` minus the address